de/css/README.md

```diff
@@ -16,7 +16,7 @@ Es wurde ursprünglich von ProgrammiererInnen bei Twitter geschrieben. Heute wir
 
 ## Bootstrap installieren
 
-Um Bootstrap zu installieren, musst du Folgendes in den `<head>` deiner `.html`-Datei einfügen:
+Öffne dein `.html` in deinem Code-Editor und füge Folgendes zum `<head>`-Abschnitt hinzu, um Bootstrap zu installieren:
 
```

Comment:
@das-g I would go for: "Öffne deine `.html`-Datei..."

Reply:
Yeah, that's better. Will you adapt it?

Reply:
Sure!